### PR TITLE
Release Notes updates for K8ssandra 1.3.0

### PR DESCRIPTION
**Pull request by johnsmartco**
**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Comment by johnsmartco** [comment]
Converted this PR to Draft - wait for K8ssandra 1.3.0 release.

**Comment by johnsmartco** [comment]
Ready for review now, but will wait for ok from @jdonenine and @bradfordcp w.r.t. 1.3.0 release before merging to main / publish.

File: docs/content/en/release-notes/_index.md
Comment (johnsmartco) [reply docs/content/en/release-notes/_index.md]: Updated - thanks Jeff.

File: docs/content/en/release-notes/_index.md
Comment (johnsmartco) [reply docs/content/en/release-notes/_index.md]: Updated - thanks Jeff.

**Review by jdonenine — APPROVED** [review]
Looks good, thanks!

**Pull request merged**
